core/src/main/resources/filodb-defaults.conf

```diff
@@ -35,6 +35,9 @@
 
       # Maximum number of chunks to flush at one time using one Unlogged Batch.
       chunk-batch-size = 16
+
+      # Maximum number of partitions can be fetched at once using IN query
```

Comment:
I think we need to expand the explanation here.  This is the max # of FiloDB partitions that can be fetched at once that will still fit in one Spark partition/thread, right?